resources/views/hardware/labels.blade.php

```diff
@@ -118,6 +118,11 @@
             <br>
         @endif
         </div>
+	<div class="pull-left">
+        @if (($settings->labels_display_company_name=='1') && ($asset->company_id!='0') && (!(is_null($asset->company_id))) && (\App\Models\Company::find($asset->company_id)->name!=''))
```

Comment:
Could we maybe just do something like:

`@if (($settings->labels_display_company_name=='1') && ($asset->company))`

?

If the company isn't valid or if the asset doesn't have one, that should (I'd think) all be covered there.